packages/pelagos/src/components/Breadcrumb.stories.js

```diff
@@ -1,27 +1,61 @@
+import MenuItem from '../menu/MenuItem';
+
 import Breadcrumb from './Breadcrumb';
 import BreadcrumbItem from './BreadcrumbItem';
+import IconMenu from './IconMenu';
 
 export default {
 	title: 'Components/Breadcrumb',
 	component: Breadcrumb,
-	decorators: [
-		(Story) => (
-			<div style={{marginLeft: '24px'}}>
-				<Story />
-			</div>
-		),
-	],
 };
 
 export const Default = {
+	args: {
+		children: [
+			<BreadcrumbItem key="a" href="#">
+				First
+			</BreadcrumbItem>,
+			<BreadcrumbItem key="b" href="#">
+				Second
+			</BreadcrumbItem>,
+		],
+	},
+};
+
+export const WithOverflowMenu = {
+	args: {
+		children: [
+			<BreadcrumbItem key="a" href="#">
+				First
+			</BreadcrumbItem>,
+			<BreadcrumbItem key="b" href="#">
+				Second
+			</BreadcrumbItem>,
+			<BreadcrumbItem key="c">
+				<IconMenu aria-label="Overflow menu">
+					<MenuItem text="Third" />
+					<MenuItem text="Fourth" />
```

Comment:
It might be useful to make these links as well, just so we know the proper way of doing that. My guess is I would do:
```
<MenuItem><a href="#">Third</a></MenuItem>
```
But I am not positive if that is correct accessibility wise.

Also should these menu items have `key` props?